src/libs/ReportActionsUtils.js

```diff
@@ -170,6 +170,28 @@ function getMostRecentIOURequestActionID(reportActions) {
     return _.last(sortedReportActions).reportActionID;
 }
 
+/**
+ * Returns array of links inside given report
+ *
+ * @param {reportAction} reportAction
+ * @returns {Boolean}
+ */
+function getLinksInsideReport(reportAction) {
```

Comment:
Could we change the name to something like `extractLinksFromMessageHtml()`. We are focusing on a single report action, so let's narrow it down vs implying we're getting all links for the whole chat.